GetIntoTeachingApi/Models/BaseModel.cs

```diff
@@ -268,32 +262,6 @@ private void MapFieldAttributesToEntity(Entity entity)
             }
         }
 
-        private void MapRelationshipAttributesToEntity(Entity source, ICrmService crm, OrganizationServiceContext context)

```

Comment:
Might be worth putting a note in the readme to say that we only map relationships _from_ the CRM to our models and not vice-versa https://github.com/DFE-Digital/get-into-teaching-api#adding-relationships